frontend/web/src/app/services/physical-print/physical-print.service.ts

```diff
@@ -36,6 +36,14 @@ export class PhysicalPrintService {
           return data;
       });
   }
+
+  uploadSlmFile(physicalPrint: PhysicalPrint, slmFile: any): Observable<string> {
+    return this.client.post(this.endpoint + '/' + physicalPrint.id + '/slm', slmFile)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a template literal instead of concatenating with a string literal. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=typescript%3AS3512)